docs/run_validator.md

```diff
@@ -0,0 +1,434 @@
+# Join Cyberd devnet as a Validator
+
+**Note** The current active dev testnet is `euler-dev` (substitute <testnet_chain_id> with that value, do not forget to remove the `<` and the `>` symbols).
+
+## Prepare your server
+
+First, you have to setup a server.
+You should to run your validator node all time. This means that you will need a reliable server to keep it running.
+Also, you may consider to use any cloud service with dedicated GPU, like Hetzner (or a local machine).
+
+Cyberd is based on Cosmos SDK and written in Go.
+It should work on any platform which can compile and run programs in Go.
+However, we strongly recommend running the validator node on a Linux server.
+
+Rank calculation in cyberd is benefit to GPU computation. 
+They are easy to parallelize. This is why it is best to use GPU.
+
+Recommended requirements:
+
+```js
+CPU: 6 cores
+RAM: 32 GB
+SSD: 256 GB
+Connection: 100Mb, Fiber, Stable and low-latency connection
+GPU: nvidia GeForce CUDA compatibale card with 6gb of memory
```

Comment:
Please describe Nvidai others GPUs familes and add to recommendation setup. Not only GeForce

Reply:
fixed